### ISSUE TITLE
Auto-save work logs, or at least warn before navigating away

### ISSUE THREAD

**Issue by emsltek** [TheHive-Project/TheHive#1391]
## Request Type
Bug / Feature Request

Auto-save work logs such that if you navigate away without saving, it auto-saves. Or, at the very least, raise a browser alert saying "unsaved form info, are you sure you want to leave?"

**Comment by gimmic** [comment]
Agreed. This could be seen as related/duplicate to https://github.com/TheHive-Project/TheHive/issues/1379

Navigation away from a changed form element should alert for data loss throughout the UI.
